### DISCUSSION TITLE
Is any version currently working?

### DISCUSSION THREAD

**Discussion by RandomUserDave** [UltimaHoarder/UltimaScraper#1582] [Q&A]
Haven't scraped in a while and all my attempts today failed.  7.4.1 has that await error that breaks the script, 7.3 runs but doesn't download anything.  I can't tell if my config or auth or something else is bad or if the script is just not working for anyone.

**Comment by RandomUserDave** [comment]
Answering myself bc I got 7.3 to work.  Had an old config, apparently we changed the auto_media_choice options at some point.

**Reply by junjuiceee** [reply]
Thanks for letting me know 7.3 works for you. I'll try that later. Have basically tried every version but that one
